### DISCUSSION TITLE
CMI side storyline

### DISCUSSION THREAD

**Discussion by TheGiraffe3** [endless-sky/endless-sky#10837] [Ideas]
EDIT: This is probably going to be included in a main storyline. Ignore it for now.

<details>
There's been a plugin floating around for a while about helping a small mining startup.

In the storyline, Core Mining Incorporated grows and grows, faces rebels in the company, conquers pirate planets, and buys out Megaparsec.
Apparently it was originally an alternate storyline (separate from FW) but I don't plan to go that far.
The plugin has been mostly abandoned; the README says the original author (@DJF113) may be working on it very slowly and will leave the repository up in case there's someone interested.

I'm thinking about cleaning it up (such as removing the selling of Megaparsec, especially not to conflict with Syndicate Business) and making a pull request. Thoughts?
https://github.com/TheGiraffe3/Core-Mining-Inc (this link will now 404)
</details>

**Comment by ziproot** [comment]
The plugin seems interesting because on the surface it looks to be completely independent of the main plot, but if you look closely, you will notice that over time events from Free Worlds happen. That seems to fit nicely with the war reactions conversation where the miners barely know what is going on, and I was actually considering a fourth mining branch. However, an issue I see is that certain characters, especially Danforth and one of the main characters in Hai Reveal, need to be met in the main plot. Even my “neutrality” plugin has missions where you fight off some pirates for Danforth and talk to two characters from war reactions, who introduce you to Alondo before transitioning to regular Hai Reveal. How hard do you think it would be to weave Danforth and a Hai Reveal character (Alondo, Remington, Xilin, Sayari, Terry) in?

**Reply by TheGiraffe3** [reply]
I was wondering about this; making an entire main storyline in Syndicate space but separate from the "real" Syndicate one seems a little much, maybe?
So I was thinking of converting this into more of a side storyline.
But it would be 100% doable to make it a main storyline, I'm just not sure whether it should be.

**Reply by ziproot** [reply]
Yeah, I think it would work well as a side storyline, especially since there aren’t many storylines in Syndicate space at the moment. @SpearDane thoughts?

**Reply by TheGiraffe3** [reply]
In particular I was wondering about stopping before the Rebels rise, as the branch where you ally with John Summers wasn't written ~~and I'm not sure I'm up to writing a branch of a major storyline in addition to the Jibauni~~.
